tensorflow_addons/losses/tests/kappa_loss_test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for Weighted Kappa loss."""
+
+import numpy as np
+from tensorflow_addons.losses.kappa_loss import WeightedKappaLoss
+
+
+def weighted_kappa_loss_np(y_true, y_pred, weightage="quadratic", eps=1e-6):
+    """Implemented in non-optimized python code to avoid mistakes"""
+
+    num_samples, num_classes = y_true.shape
+    numerator = 0
+    true_classes = y_true.argmax(axis=1)
+    for i in range(num_samples):
+        true_class = true_classes[i]
+        for j in range(num_classes):
+            if weightage == "quadratic":
+                w_ij = np.power(j - true_class, 2)
+            else:
+                w_ij = np.abs(j - true_class)
+            numerator += w_ij * y_pred[i, j]
```

Comment:
Would it be possible to verctorize those? I believe we would gain in readability.